frontend/micro-ui/web/micro-ui-internals/packages/modules/Project/src/pages/employee/CreateProject/CreateProjectResponse.js

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect, useState } from "react";
+import { Link, useHistory, useLocation } from "react-router-dom";
+import { useTranslation } from "react-i18next";
+import { Banner, Card, CardText, LinkLabel, EditIcon} from "@egovernments/digit-ui-react-components";
+import useQueryParams from "../../../../../../libraries/src/hooks/useQueryParams";
```

Comment:
you shouldn't import outside a package.
@hariomsinha-eGov

Reply:
sure Jagan. I should have used the hook to call the hook.